src/main/resources/assets/nei/csv/catalysts.csv

```diff
@@ -1316,4 +1320,24 @@ crafting,BiblioCraft:BiblioWorkbench:4,,BiblioCraft,TRUE,,,,,,
 crafting,BiblioCraft:BiblioWorkbench:5,,BiblioCraft,TRUE,,,,,,
 crafting,BiblioCraft:BiblioWorkbench:6,,BiblioCraft,TRUE,,,,,,
 crafting,ComputerCraft:CC-TurtleExpanded,{leftUpgrade:2},ComputerCraft,TRUE,,,,,,
-crafting,ComputerCraft:CC-TurtleAdvanced,{leftUpgrade:2},ComputerCraft,TRUE,,,,,,
\ No newline at end of file
+crafting,ComputerCraft:CC-TurtleAdvanced,{leftUpgrade:2},ComputerCraft,TRUE,,,,,,
+crafting,gregtech_addon:machine:16,,gregtech_addon,TRUE,,,,,,
```

Comment:
We already have some catalysts for GT4. Can you move these to there?

Reply:
This is [Gregtech 4](https://www.curseforge.com/minecraft/mc-mods/gregtech-4)

Reply:
Sure thing